.github/workflows/release-suite-native-production.yml

```diff
@@ -125,7 +125,39 @@ jobs:
           BUILD_URL=$(eas build:view "$BUILD_ID" --json | jq -r '.artifacts.buildUrl')
           echo "build_url=$BUILD_URL" >> $GITHUB_OUTPUT
 
-      - name: Upload APK to S3
+      - name: Download APK
         run: |
           wget -O "Trezor-Suite-${{ steps.get_version.outputs.app_version }}.apk" "${{ steps.eas_build.outputs.build_url }}"
-          aws s3 cp "./Trezor-Suite-${{ steps.get_version.outputs.app_version }}.apk" s3://data.trezor.io/suite/releases/mobile/
+
+      - name: Upload APK as workflow artifact
```

Comment:
This is probably an extra step, but can be useful to see APK as artifact in the github?

Reply:
We're adding it to the release in this exact job. Why have it duplicated? It isn't necessary.

Reply:
removed 52377945ad7d5e59bd2dda22035765a9ef0510b0